Classes/Service/ListService.php

```diff
@@ -37,12 +37,17 @@ public static function resolveListitems(array &$row, $field = 'tx_listelements_l
             $returnAs = 'listitems';
         }
 
+        $workspace = 0;
+        if (isset($GLOBALS['BE_USER']) && isset($GLOBALS['BE_USER']->workspace)) {
```

Comment:
In  general we should use the Context API
```GeneralUtility::makeInstance(Context::class)->getAspect('workspace')->getId()``` which always works

Reply:
Right. I'll change it 👍🏻